test/test_python_client.py

```diff
@@ -218,4 +218,33 @@ def test_route_argument_doc_string(self):
         self.assertEqual(backend._format_type_in_doc(ns, Map(String(), Int32())),
                          'Map[str, int]')
 
+    def test_route_arbitrary_attrs(self):
+        # type: () -> None
+
+        route1 = ApiRoute('get_metadata', 1, None)
+        route1.set_attributes(None, None, Void(), Void(), Void(), {'scope':'events.read'})
+        ns = ApiNamespace('files')
+        ns.add_route(route1)
+
+        result = self._evaluate_namespace(ns)
+        expected = textwrap.dedent('''\
+            def files_get_metadata(self):
+                """
+                Route attributes:
+                    scope: events.read
```

Comment:
What is the expected output for multiple attributes selected? Is it inline or new lines?

Reply:
New lines. I'll add more testing.